### ISSUE TITLE
`chromatic` parameter not available in FSIM loss

### ISSUE THREAD

**Issue by zakajd** [photosynthesis-team/piq#145]
**Describe the bug**
During initial PR #109 I forgot to add `chromatic` parameter` to the loss. So now only greyscale option is supported.

**To Reproduce**

1. `from piq import FSIMLoss`
2. `loss = FSIMLoss(chromatic=True)`

**Expected behavior**
Same features should be available in FSIMLoss as in `fsim` function.
